internal/command/status/machines.go

```diff
@@ -135,7 +136,8 @@ func renderMachineStatus(ctx context.Context, app *api.AppCompact) error {
 			msgs = append(msgs, msg)
 		}
 
-		fmt.Fprintln(io.Out, colorize.Yellow(strings.Join(msgs, "")))
+		fmt.Fprintln(out, colorize.Yellow(strings.Join(msgs, "")))
+		// TODO: receive an error output writer as an argument
 		fmt.Fprintln(io.ErrOut, colorize.Yellow("Run `flyctl image update` to migrate to the latest image version."))
```

Comment:
Not sure whether to keep `io.ErrOut` here or use the same `out` channel here. It all goes to the terminal in the end.